ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js

```diff
@@ -158,26 +158,40 @@ describe("Form with File Input - Prefill & Submit tests", () => {
         })
     };
 
+    const deleteSelectedFiles = (component, fileNames) => {
+        cy.get(component).then(() => {
+            fileNames.forEach((fileName) => {
+                cy.get("[aria-label='" + fileName + "']").next().click();
```

Comment:
Any CSS selector used should be BEM, aria-label is not part of the contract

Reply:
okay